service/src/test/java/com/sedliarov/learningtable/integration/StudentControllerIntegrationTests.java

```diff
@@ -42,6 +46,9 @@ public class StudentControllerIntegrationTests extends RestIntegrationTestBase {
   @Autowired
   private StudentMapper mapper;
 
+  @Autowired
+  private MessageService messageService;
+
```

Comment:
You can define static String messages. We don't need this service

Reply:
Done